samples/getallvms.py

```diff
@@ -77,6 +77,11 @@ def main():
     args = cli.get_args()
 
     try:
+        if(args.no_ssl == 1):
```

Comment:
As stated in https://www.python.org/dev/peps/pep-0476/, disable the certificate checking globally is discouraged. We should do something like following:
context = hasattr(ssl, '_create_unverified_context') and ssl._create_unverified_context() \
          or None
then pass the context to Connect and SmartConnect methods.